### ISSUE TITLE
IMA agent collects 16 BLS-sign parts even collected 11 threshold number of BLS signature

### ISSUE THREAD

**Issue by PolinaKiporenko** [skalenetwork/IMA#1429]
https://github.com/skalenetwork/internal-support/issues/265

**Comment by DmytroNazarenko** [comment]
[2.0.0-beta.0](https://github.com/skalenetwork/IMA/releases/tag/2.0.0-beta.0)

**Comment by EvgeniyZZ** [comment]
Verified: New IMA Agent not wait all 16 responses. Boundary case was not tested. Temporary blocked by another issue. 
IMA Agent: 2.0.0-beta.13

[imaVerifyAndSignParallelB13.txt](https://github.com/skalenetwork/IMA/files/12665120/imaVerifyAndSignParallelB13.txt)
